### ISSUE TITLE
Rolling builds branch filtering

### ISSUE THREAD

**Issue by IlyaFinkelshteyn** [appveyor/ci#1966]
Customer can have specific branches where rolling builds are useful, while in other branches they are harmful.

Implement together with https://github.com/appveyor/ci/issues/1878

**Comment by paddie** [comment]
Yes, maybe support specifying a blacklist/whitelist of branches that should not be pre-empted. We need every merge to `master` to result in a release, but every PR can essentially be pre-empted.

In travis-ci, there's the option to just enable rolling builds for PRs and we'd love to see that option in AppVeyor.

Thank you.

**Comment by IlyaFinkelshteyn** [comment]
@paddie but https://github.com/appveyor/ci/issues/2066 is implemented couple of month ago...
